src/network/tarcap/packets_handler/handlers/test_packet_handler.cpp

```diff
@@ -8,6 +8,12 @@ TestPacketHandler::TestPacketHandler(std::shared_ptr<PeersState> peers_state, co
 void TestPacketHandler::process(const PacketData& packet_data, const dev::RLP& packet_rlp) {
   assert(packet_data.type_ == SubprotocolPacketType::TestPacket);
 
+  std::cout << "TestPacketHandler::process" << std::endl;
+  std::cout << std::string(packet_rlp[0]) << std::endl;
+  std::cout << std::string(packet_rlp[1]) << std::endl;
+
+  return;
```

Comment:
This looks like temporary code?

Reply:
aah yes I was testing if it works with some dummy data and forgot to delete it, thanks :)